openapi/ip2whois/openapi.yaml

```diff
@@ -0,0 +1,55 @@
+openapi: 3.0.0
+servers:
+  - url: https://api.ip2whois.com/v1
+info:
+  description: >
+    This is a generated connector from [IP2WHOIS API v2](https://www.ip2whois.com/developers-api) OpenAPI Specification. 
+
+    IP2WHOIS is a free tool to allow you to check WHOIS information for a particular domain, such as domain assigned owner contact information, registrar information, registrant information, location and much more.
+  x-init-description: >
+    The connector initialization requires setting the API credentials.
+
+    Create a [IP2WHOIS Account](https://www.ip2whois.com/register) and obtain the API token by navigating to the [IP2WHOIS Profile](https://www.ip2whois.com/login)
+  title: IP2WHOIS Domain Lookup
+  version: "1.0"
+  x-apisguru-categories:
+    - developer_tools
+  x-origin:
+    - format: openapi
+      url: https://api.swaggerhub.com/apis/ip2whois/ip2whois-domain-lookup/1.0
+      version: "3.0"
+  x-providerName: ip2whois.com
+paths:
+  /:
+    get:
+      operationId: "getDomainInfo"
+      description: Lookup WHOIS information
```

Comment:
Can you change the `description` to `summary`? Then it will add a description in the function doc comment.